app/services/riiif/imagemagick_command_factory.rb

```diff
@@ -39,7 +43,7 @@ def build
     private
 
       def command
```

Comment:
Yes, I think we should avoid redirection where we don't need it. This is a private method so there should be very little impact in making the change.